client/src/Constants.js

```diff
@@ -0,0 +1,10 @@
+/* Store constant values here */
```

Comment:
constant names are usually all upper-case

Reply:
I know I was thinking about that afterwards. I will change it at some point